test/libyul/yulOptimizerTests/loadResolver/keccak_reuse_reassigned_branch.yul

```diff
@@ -0,0 +1,30 @@
+{
+    let x := calldataload(0)
+    let y := calldataload(1)
+    let a := keccak256(x, y)
+    if calldataload(2) {
+        a := 8
+    }
+    let b := keccak256(x, y)
+    sstore(b, 2)
+    if calldataload(3) {
+        x := 8
```

Comment:
wouldn't hurt to do the same thing with only ``y`` instead of only ``x``, but yeah, also not crucial...